polkadot/runtime/parachains/src/configuration/migration/v12.rs

```diff
@@ -0,0 +1,349 @@
+					max_head_data_size                       : pre.max_head_data_size,
+					max_upward_queue_count                   : pre.max_upward_queue_count,
+					max_upward_queue_size                    : pre.max_upward_queue_size,
+					max_upward_message_size                  : pre.max_upward_message_size,
+					max_upward_message_num_per_candidate     : pre.max_upward_message_num_per_candidate,
+					hrmp_max_message_num_per_candidate       : pre.hrmp_max_message_num_per_candidate,
+					validation_upgrade_cooldown              : pre.validation_upgrade_cooldown,
+					validation_upgrade_delay                 : pre.validation_upgrade_delay,
+					max_pov_size                             : pre.max_pov_size,
+					max_downward_message_size                : pre.max_downward_message_size,
+					hrmp_sender_deposit                      : pre.hrmp_sender_deposit,
+					hrmp_recipient_deposit                   : pre.hrmp_recipient_deposit,
+					hrmp_channel_max_capacity                : pre.hrmp_channel_max_capacity,
+					hrmp_channel_max_total_size              : pre.hrmp_channel_max_total_size,
+					hrmp_max_parachain_inbound_channels      : pre.hrmp_max_parachain_inbound_channels,
+					hrmp_max_parachain_outbound_channels     : pre.hrmp_max_parachain_outbound_channels,
+					hrmp_channel_max_message_size            : pre.hrmp_channel_max_message_size,
+					code_retention_period                    : pre.code_retention_period,
+					max_validators                           : pre.max_validators,
+					dispute_period                           : pre.dispute_period,
+					dispute_post_conclusion_acceptance_period: pre.dispute_post_conclusion_acceptance_period,
+					no_show_slots                            : pre.no_show_slots,
+					n_delay_tranches                         : pre.n_delay_tranches,
+					zeroth_delay_tranche_width               : pre.zeroth_delay_tranche_width,
+					needed_approvals                         : pre.needed_approvals,
+					relay_vrf_modulo_samples                 : pre.relay_vrf_modulo_samples,
+					pvf_voting_ttl                           : pre.pvf_voting_ttl,
+					minimum_validation_upgrade_delay         : pre.minimum_validation_upgrade_delay,
+					async_backing_params                     : pre.async_backing_params,
+					executor_params                          : pre.executor_params,
+					minimum_backing_votes                    : pre.minimum_backing_votes,
+					node_features                            : pre.node_features,
+					approval_voting_params                   : pre.approval_voting_params,
+					scheduler_params: SchedulerParams {
+							group_rotation_frequency             : pre.group_rotation_frequency,
+							paras_availability_period            : pre.paras_availability_period,
+							max_validators_per_core              : pre.max_validators_per_core,
+							lookahead                            : pre.scheduling_lookahead,
+							coretime_cores                       : pre.coretime_cores,
```

Comment:
Another nit: Given that coretime cores are now pretty much just cores, we could rename this to cores or num_cores.

Reply:
Done